### PR TITLE
change binomial link

### PR DESCRIPTION
**Pull request by MikeJSeo**

### CODE REVIEW

File: R/maic_anchored.R [hunk 1]
```diff
@@ -485,8 +485,8 @@ maic_anchored_binary <- function(res,
   # ~~~ Analysis table
   # : set up proper link
   glm_link <- switch(eff_measure,
-    "RD" = poisson(link = "identity"),
-    "RR" = poisson(link = "log"),
+    "RD" = binomial(link = "identity"),
```
Comment (gravesti) [inline R/maic_anchored.R:488]: @MikeJSeo From the checks I see that `binomial("identity")` is not valid.

**Comment by gravesti** [comment]
@MikeJSeo Does this change anything? Or do our tests not cover these results?

File: R/maic_anchored.R (hunk 1 above)
Comment (MikeJSeo) [reply R/maic_anchored.R:488]: @gravesti 
so binomial("identity") does seem to work when the covariate is discrete (0/1 like as in treatment indicator), but doesn't work if the covariate is continuous. Would this possibly be the reason?
Which check does it fail?

Or could this be R version issue?
https://stats.stackexchange.com/questions/139917/r-binomial-family-with-identity-link

It does seem to change the results quite a bit

**Comment by MikeJSeo** [comment]
I don't know I just updated the snapshots
image

Why did it pass all these test before?

@gravesti

File: R/maic_anchored.R (hunk 1 above)
Comment (gravesti) [reply R/maic_anchored.R:488]: Looking in the code for `glm` and `binomial` I don't see why there should be a warning with out code. The link function is correctly set as identity, so let's leave that for now.

Regarding the changed results:

- I think the fix to the N makes the `$descriptive` table correct.

- The changes to the SE (and p-val) I'm not sure. I'd be pleased if @hoppanda could review...

Comment (MikeJSeo) [reply R/maic_anchored.R:488]: Hi @gravesti,
This originally started from my mistake where I thought this identity or log link doesn't work for binomial.
It turns out that for binary covariates, this works fine.
Can you merge this in for the time being? I think this is quite a big of an error that we don't want people to use in terms of SE/p-val.

**Review by gravesti — APPROVED** [review]

**Pull request merged**